components/gitpod-db/go/team.go

```diff
@@ -75,3 +75,23 @@ func GetTeamBySlug(ctx context.Context, conn *gorm.DB, slug string) (Team, error
 
 	return team, nil
 }
+
+func GetSingleTeam(ctx context.Context, conn *gorm.DB) (Team, error) {
```

Comment:
```suggestion
func ExactlyOneTeamExists(ctx context.Context, conn *gorm.DB) (Team, error) {
```
This would better communicate the semantics, as I could take `GetSingleTeam` as returning a team, not necessarily only when there is exactly one in the cell.

Reply:
DB unit test for this would be appreciated

Reply:
`ExactlyOneTeamExists` suggests a boolean type. Maybe `GetTheSingleTeam` together with docs?